css/layout-css/news-feed-style.upload.css

```diff
@@ -761,6 +761,9 @@
   position: relative;
   margin: 8px 0 30px 0;
   transition: all 0.1s;
+  word-break: break-word;
```

Comment:
Can you also check that long words are correctly broken in all LFD layouts? Please check all the fields that are displayed, e.g. title, description, categories and other labels etc.

Reply:
@tonytlwu I've checked layouts and seems only in the directory long text is working okay.
![directory long text](https://user-images.githubusercontent.com/53430352/78884099-1459c580-7a63-11ea-869c-8a414f649022.png)
![featured list](https://user-images.githubusercontent.com/53430352/78884104-158af280-7a63-11ea-83a4-3368683ed4cd.png)
![news feed long text](https://user-images.githubusercontent.com/53430352/78884105-158af280-7a63-11ea-921b-158672cbf261.png)
![simple list long text](https://user-images.githubusercontent.com/53430352/78884106-16238900-7a63-11ea-9b23-b76c6cc8a6b0.png)
![agenda long text](https://user-images.githubusercontent.com/53430352/78884109-16238900-7a63-11ea-826a-bf8ae01d2b79.png)

Should we fix all layouts in this PR?

Reply:
Yes. Please fix them all.

Reply:
@tonytlwu 
corrected all layouts how they look now https://share.getcloudapp.com/NQuDEgzX